app/services/message_sender.rb

```diff
@@ -15,12 +16,19 @@ def run
         client.chat_postMessage(
           channel: channel_id,
           as_user: true,
-          text: message.body
+          text: escaped_message_body
         )
       end
     end
   end
 
+  def escape_specific_characters(message_body)
```

Comment:
Yeah, likely a class in `app/services` that takes a message body as an argument and returns the message encoded. Would basically just include the logic in this method, and would allow us to write some AWESOME unit tests (otherwise, we'll have to test via `MessageSender` which seems kinda messy)